.github/workflows/start-release.yml

```diff
@@ -209,15 +209,22 @@ jobs:
         uses: actions/checkout@v3
         id: checkout
         with:
+          submodules: true
           ref: "${{ needs.release-setup.outputs.release-branch }}"
           token: "${{ steps.token.outputs.token }}"
-      - name: Create release version tag
+      - name: Create semgrep release version tag
         id: create-tag
         run: |
           git config user.name ${{ github.actor }}
           git config user.email ${{ github.actor }}@users.noreply.github.com
           git tag -a -m "Release ${{ needs.release-setup.outputs.version }}" "v${{ needs.release-setup.outputs.version }}"
           git push origin "v${{ needs.release-setup.outputs.version }}"
+      - name: Create semgrep-interfaces release version tag
+        id: create-tag
```

Comment:
@ajbt200128 @spencerdrak this broke the release, because we have 2 steps with the same id.
Why do we need an id in the first place anyway?
cc @mjambon

Reply:
It's' used for referencing the output of the steps later - I wonder if we can write a semgrep rule to check for duplicate step IDs. If you don't reference the step later, the id can be omitted, although I am in the habit of always adding them.